transcriptic/jupyter/objects.py

```diff
@@ -501,6 +512,147 @@ def _repr_html_(self):
                self.connection.get_route('view_data', data_id=self.id)
 
 
+class DataObject(object):
+    """
+    A DataObject holds a reference to the raw data, stored in S3, along with format and validation information
+
+    Attributes
+    ----------
+    id : str
+        DataObject id
+    dataset_id : str
+        Dataset id
+    data : bytes
+        Bytes fetched from the url
+    name: str
+        Dataset name
+    content_type: str
+        content type
+    format: str
+        format
+    size: int
+        size in bytes
+    status: string
+        valid vs invalid
+    url: str
+        download url which expires every 1hr.  Call `refresh` to renew
+    validation_errors: Array[str]
+        validation errors
+    container: Container
+        Container object that was used for this data object
+    aliquot: Aliquot
+        Aliquot object that was used for this data object
+    attributes: dict
+        Master attributes dictionary
+    connection: transcriptic.config.Connection
+        Transcriptic Connection object associated with this specific object
+    """
+
+    def __init__(self, id=None):
+        attributes = {}
+
+        # Fetch dataobject from server if id supplied
+        if id is not None:
+            attributes = DataObject.fetch_attributes(id)
+
+        self.__init_attrs(attributes)
+
+        # cached values
+        self._container = None
+        self._data      = None
+        self._json      = None
+
+    def __init_attrs(self, attributes):
+        self.attributes = attributes
+
+        self.id                = attributes.get('id')
```

Comment:
nit: spacing.. this looks fine, but yea, would prefer consistency with the other spacing formatting